### ISSUE TITLE
Suspender and warning

### ISSUE THREAD

**Issue by CJ-Wright** [xpdAcq/mission-control#93]
We may need a suspender and warning for a closed upstream shutter. Do we have a PV for the upstream shutter?

## Steps to Reproduce (for bugs)

1.
2.
3.

**Comment by CJ-Wright** [comment]
@sghose

**Comment by CJ-Wright** [comment]
`XF:29IDA-PPS{PSh}Pos-Sts` is the positional status for the A hutch shutter/

**Comment by chiahaoliu** [comment]
Would it be possible to configure it as an ophyd object?

**Comment by CJ-Wright** [comment]
Yes we need an ophyd object, I was just putting the PV here so we could keep track of it

**Comment by CJ-Wright** [comment]
@xpdAcq/bls has there been any progress with this?

**Comment by sbillinge** [comment]
@DanOlds this just needs an ophyd object for the beam-shutter but would be really useful functionality such as starting scans before the hutch is closed, but having the scan not _actually_ start until it detects the beam-shutter is open.   The DAMA group can (has?) train how to make Ophyd objects, but it is much better if the beamline staff can take responsibility for actually writing them.....
